frontend/src/app/game/components/speed-control/speed-control.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-button-toggle-group
+  #group grow
+  [value]="value"
+  (change)="onUserAction($event.value)"
+>
+  <mat-button-toggle
+    *ngFor="let level of levels"
```

Comment:
Please, add matTooltip with short descriptions of the purpose. Some of the icons do not have to be crysta clear to everybody. @JaroslavSoldat